apps/passport-client/src/useSyncE2EEStorage.tsx

```diff
@@ -151,18 +160,50 @@ export async function uploadSerializedStorage(
       success: true,
       value: { revision: uploadResult.value.revision, storageHash: storageHash }
     };
+  } else if (uploadResult.error.name === "Conflict") {
+    console.warn("[SYNC] conflict uploading e2ee storage", uploadResult.error);
+    return { success: false, error: uploadResult.error };
   } else {
     console.error("[SYNC] failed to upload e2ee storage", uploadResult.error);
     return { success: false, error: uploadResult.error };
   }
 }
 
+export type MergeState = {
+  pcds: PCDCollection;
+  subscriptions: FeedSubscriptionManager;
+};
```

Comment:
not sure what the purpose of this type is from just looking at it. is `merge` a verb or a noun here? what is the context of `state` - `AppState` or something else?

Reply:
Merge is meant to be a noun, and state is meant to be as generic as possible because I didn't have a good name.  I'll try again.